# executable in /usr/games
# share dir in /usr/share/games/evernus (lower case)
Index: evernus-1.33/CMakeLists.txt
===================================================================
--- evernus-1.33.orig/CMakeLists.txt	2015-07-27 21:20:11.459537278 +0200
+++ evernus-1.33/CMakeLists.txt	2015-07-27 21:20:11.455537204 +0200
@@ -777,7 +777,7 @@
 
 set(RESOURCES
     "res"
-    "trans"
+    "${CMAKE_CURRENT_BINARY_DIR}/trans"
 )
 
 if(APPLE)
@@ -804,8 +804,8 @@
     )
 elseif(UNIX)
     foreach(RES ${RESOURCES})
-        install(DIRECTORY ${RES} DESTINATION "/usr/share/Evernus")
+        install(DIRECTORY ${RES} DESTINATION "/usr/share/games/evernus")
     endforeach()
 
-    install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION "/usr/bin")
+    install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION "/usr/games")
 endif()
